index-2.html

```diff
@@ -0,0 +1,181 @@
+          wikipedians, Open knowledge, galleries, libraries, museums, archives,
+          governments and foundations, lawyers, and activists - we hope you'll
+          consider joining us this year to build a stronger, more vibrant
+          commons together
+        </p>
+      </div>
+      <div class="last-p">
+        <p>
+          Please contact us per Email for nay further questions about CC Global
+          Summit 2015!
+        </p>
+        <h3>ccsummit2015@cckorea.org</h3>
+      </div>
+    </section>
+    <!-- creative -->
+    <section class="creative">
+      <div class="creative-div">
+        <h3 class="creative-p">Creative Commons Global Summit 2015 Logo</h3>
+        <hr />
+        <p class="creative-p2">
+          The logo of CC Global Summit 2015 was decided through a competition in
+          may.
+        </p>
+        <p class="alternate-creative-p">
+          The logo of CC Global Summit 2015 was decided through the logo
+          competition from 8, June to 7, July. After online public voting and
+          Selection Committee’s judging process, the logo from Naresh Agrawal
+          won the prize. After consultation with Mr. Agrawal, we optimized the
+          prize-winning-logo for this year’s summit as follows:
+        </p>
+      </div>
+      <div class="late-logo">
+        <p class="repeat-logo">
+          <a class="food-1" href="./index.html">
+            <img class="food" src="./picture/bird2.jpg" alt="Bird House"
+          /></a>
+          <a href="./index.html" class="logo"> Bird House</a>
+        </p>
+      </div>
+      <hr class="full-hr" />
+    </section>
+    <section class="last-sectinon">
+      <div class="creative-div">
+        <h3 class="creative-p">See the past CC Global Summits</h3>
+        <hr />
+        <p class="creative-p22">
+          Take a look at the past two CC Global Summits which took place in
+          Buenos Aires and in Warsaw
+        </p>
+      </div>
+      <div class="last-pic">
+        <img
+          src="./picture/about_2011 1.png"
+          alt=" CC Global Summit 2011 in Warsaw"
+        />
+        <img
+          src="./picture/about_2013 1.png"
+          alt="CC Global Summit 2013 in Buenos Aires"
+        />
+      </div>
+      <div class="creative-div3">
+        <h3 class="creative-p3">Partner</h3>
+        <hr />
+        <div class="last-link">
+          <img src="./picture/mozilla 1.png" alt="www.mozilla.com" />
+          <img
+            src="./picture/googlekorea 1.png"
+            alt="https://www.google.co.kr/"
+          />
+          <img src="./picture/naver 1.png" alt="www.naver.com" />
+          <img
+            src="./picture/daumkakaologo 1.png"
+            alt="www.daumkakaologo.com"
+          />
+          <img src="./picture/airbnb 1.png" alt="www.airbnb.com" />
+        </div>
+      </div>
+    </section>
+    <div class="footer">
+      <img src="./picture/fin.png" alt="creative commonns korea" />
```

Comment:
- [ ] remember you are required to personalize your content but follow the design guidelines. Please use your bird logo here, and you can write your own text beside as well. This helps you to be creative and your work to be unique